base/src/main/java/io/spine/validate/MessageValidator.java

```diff
@@ -201,8 +207,11 @@ public void visitRequiredOneof(IsRequiredConstraint constraint) {
             ConstraintViolation violation = ConstraintViolation
                     .newBuilder()
                     .setMsgFormat(constraint.errorMessage(message.context()))
-                    .setFieldPath(Field.named(constraint.oneofName().value()).path())
-                    .setTypeName(targetType.name().value())
+                    .setFieldPath(Field.named(constraint.oneofName()
+                                                        .value())
+                                       .path())
```

Comment:
Please realign this fragment. You could either extract a variable to preserve the consistency with automatic formatting or simply put `constraint.oneofName().value()` into a single line.

I assume there's no way to match the type of the param to the type of the return value.